addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1965,6 +1965,34 @@ options.registry.topMenuColor = options.Class.extend({
     },
 });
 
+options.registry.MobileVisibility = options.Class.extend({
+    isTopOption: true,
+
+    //--------------------------------------------------------------------------
+    // Public
+    //--------------------------------------------------------------------------
+
+    /**
+    * @override
+    */
+    async start() {
+        this._super(...arguments);
+        this.$el.find('we-button').attr('title', this.$target.hasClass('d-md-block d-none') ? _t("Hidden on mobile") : _t("Visible on mobile"));
+    },
+
+    //--------------------------------------------------------------------------
+    // Options
+    //--------------------------------------------------------------------------
+
+    /**
+    * @override
+    */
+    selectClass(previewMode, widgetValue, params) {
+        this._super(...arguments);
```

Comment:
Same here, the parent is async, this should be `await`ed (whatever or not there is actual async part: our framework can be extended)